crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -77,27 +80,31 @@ def create(self, vals):
             }
         )
         res.event_id = event
-        res.add_owner2participants()
+        # res.add_owner2participants()
         return res
 
-    @api.multi
-    def write(self, vals):
-        super().write(vals)
-        self.add_owner2participants()
-        return True
-
-    @api.multi
-    def add_owner2participants(self):
-        """Add the project owner to the participant list. """
-        for project in self:
-            if project.project_owner_id not in project.participant_ids.mapped(
-                "partner_id"
-            ):
-                participant = {
-                    "partner_id": project.project_owner_id.id,
-                    "project_id": project.id,
-                }
-                project.write({"participant_ids": [(0, 0, participant)]})
+    # TODO: add_owner2participants is commented out for now as it creates duplicate
+    # project owners, check when project creation form is done how to best
+    # link project, owner and participants
+
+    # @api.multi
+    # def write(self, vals):
+    #     super().write(vals)
+    #     self.add_owner2participants()
+    #     return True
+
+    # @api.multi
+    # def add_owner2participants(self):
+    #     """Add the project owner to the participant list. """
+    #     for project in self:
+    #         if project.project_owner_id not in project.participant_ids.mapped(
+    #             "partner_id"
+    #         ):
+    #             participant = {
+    #                 "partner_id": project.project_owner_id.id,
+    #                 "project_id": project.id,
+    #             }
+    #             project.write({"participant_ids": [(0, 0, participant)]})
```

Comment:
I think rather than commenting this, you should remove at the other place if it was done manually elswhere to create the owner participant.

Reply:
Ok